src/main/java/com/ldtteam/structures/client/BlueprintHandler.java

```diff
@@ -113,10 +114,10 @@ public void clearCache()
      * @param partialTicks the partial ticks.
      * @param blueprint    the blueprint.
      */
-    public void drawBlueprintAtListOfPositions(final List<BlockPos> points,
-        final float partialTicks,
-        final Blueprint blueprint,
-        final MatrixStack stack)
+    public void drawAtListOfPositions(final Blueprint blueprint,
```

Comment:
Will probably need a PR on minecolonies side too, right?

Reply:
Yep

Reply:
Don't call this, use Client Handler instead